packages/ui-components/src/components/range/range.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component, Host, h, Prop, Element, EventEmitter, Event } from '@stencil/core';
+import { IRange, IRangeEvents } from './range.types';
+import { getInputPercentageFromValue, getOffset } from './range.helper';
+
+@Component({
+	tag: 'kv-range',
+	styleUrl: 'range.scss',
+	shadow: true
+})
+export class KvRange implements IRange, IRangeEvents {
+	@Element() el!: HTMLKvRangeElement;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) min: number;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) max: number;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) value?: number = 0;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) step?: number = 1;
+	/** @inheritdoc */
+	@Event() valueChange: EventEmitter<number>;
+
+	componentDidRender() {
+		const slider = this.el.shadowRoot.getElementById('slider') as HTMLElement;
+		this.applyCssStyles();
+		slider.oninput = () => {
```

Comment:
Can you put these directly in the JSX or not? 
`<input id="slider" class="slider" type="range" min={this.min} max={this.max} value={this.value} step={this.step}  oninput={this.applyCssStyles}/>`

Reply:
@ricardogoncalves89 updated on new pr